jib-cli/src/test/java/com/google/cloud/tools/jib/cli/jar/JarModeProcessorTest.java

```diff
@@ -417,4 +423,318 @@ public void testComputeEntrypointForPackagedStandard_withMainClass()
     assertThat(actualEntrypoint)
         .isEqualTo(ImmutableList.of("java", "-jar", "/app/standardJarWithClassPath.jar"));
   }
+
+  @Test
+  public void testCreateLayersForExplodedLayeredSpringBoot()
+      throws IOException, URISyntaxException {
```

Comment:
I think it's worth adding the contents of `layer.idx` as a code comment.

Reply:
Good idea.